packages/components/FocusZone/src/FocusZone.types.ts

```diff
@@ -63,6 +64,11 @@ export type FocusZoneProps = React.PropsWithChildren<{
    * Callback called when “focus” event triggered in FocusZone
    */
   onFocus?: (e?: any) => void;
+
+  /**
+   * Allow consumers to pass in Style props
+   */
+  style?: ViewStyle;
```

Comment:
Could we just extend ViewProps too?

Reply:
At least on macOS, I think FocusZone accepts all or most View props. I never got around to changing the types though

Reply:
FocusZone doesn't actually have a View slot, so I don't think it's viable to also extend ViewProps. We just want to give the consumer access to some layout properties.

Reply:
What I meant was, macOS FocusZone extends RCTView and RCTViewManager, so the native component itself accepts most view props. I think when I asked (and we looked) at the win32 FZ's implementation, that was also true

Reply:
There's an added complexity to some of the props exposed by ViewProps as it relates to FocusZone - collapsable, focusable are some.

For now, adding Style solves the problems of some of our partners. If someone brings this up in the future, definitely open for a discussion :)

Reply:
Thanks for the callout though - it's a valid thought.